tests/test-solve.C

```diff
@@ -132,10 +151,8 @@ bool run_with_field (Givaro::Integer q, size_t b, size_t m, size_t iters, uint64
             // testing a parallel run
         std::cout<<" par: ";
         FFLAS::ParSeqHelper::Parallel<FFLAS::CuttingStrategy::Recursive,FFLAS::StrategyParameter::Threads> parH;
-        PAR_BLOCK{
```

Comment:
Ok for this change, but then, make it simpler, and replace this ParSeqHelper by a boolean since you will never use this parseq helper later on.

Reply:
How about changing that in the following way ?
        std::cout<<" par: ";
        ok = ok && check_solve(*F,m,G,
        FFLAS::ParSeqHelper::Parallel<FFLAS::CuttingStrategy::Recursive,FFLAS::StrategyParameter::Threads>());

This will use the compile time polymorphism which is more efficient I think.

Reply:
No it is just overkill to pass a ParSeqHelper just to make a boolean switch par/seq, since you don't use the helper objetc anyway.